internals/config/default.yml

```diff
@@ -0,0 +1,252 @@
+---
+name: "default name"
+slug: "default"
+
+settings:
+  mobile_phone_verification: false
+  subscriptions: false
+
+# configuration variables for the server app
+server:
+  # put the URL of openwisp-radius here
+  host: "http://127.0.0.1:8000"
+  proxy_urls:
+    password_change: "/api/v1/{org_slug}/account/password/change"
+    password_reset: "/api/v1/{org_slug}/account/password/reset"
+    password_reset_confirm: "/api/v1/{org_slug}/account/password/reset/confirm"
+    registration: "/api/v1/{org_slug}/account"
+    user_auth_token: "/api/v1/{org_slug}/account/token"
+    validate_auth_token: "/api/v1/{org_slug}/account/token/validate"
+    authorize: "/api/v1/authorize"
+    user_radius_sessions: "/api/v1/{org_slug}/account/session"
+    plans: '/api/v1/{org_slug}/plans'
+    create_mobile_phone_token: '/api/v1/{org_slug}/account/phone/token'
+    verify_mobile_phone_token: '/api/v1/{org_slug}/account/phone/verify'
+    mobile_phone_number_change: '/api/v1/{org_slug}/account/phone/change'
+  uuid: organization_uuid
+  secret_key: organization_secret_key
+  timeout: 2 #request timeout period in seconds
+
+# configuration variables for the client
+# Path of all the assets should be defined relative to the organization's folder
+# in assets directory.
+client:
+
+  auto_login: True
+
+  # path of favicon
+  favicon: "favicon.png"
+
+  # path of the custom css file relative to organization's folder in
+  # assets directory.
+  css_path: "index.css"
+  languages:
+    - text: "english"
+      slug: "en"
+  default_language: "en"
+  components:
+    header:
+      logo:
+        url: "openwisp-logo-black.svg" # logo url
+        alternate_text: "openwisp"
+      links:
+        - text:
+            en: "Sign In"
+          url: "/default/login"
```

Comment:
can we find a way to use `{org_slug}` here too? This would allow copying this part between orgs easier.